source/communication_interfaces/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ include(FindPkgConfig)
 
 add_project_dependency(control_libraries 7.0.0 REQUIRED COMPONENTS state_representation)
 add_project_dependency(clproto 7.0.0 REQUIRED)
-add_project_dependency(cppzmq 4.7.1 REQUIRED)
+if (NOT ${cppzmq_FOUND}) # provided by parent CMakeLists.txt (if used)
```

Comment:
Maybe this is a bit confusing at the moment. `network_interfaces` and `communication_interfaces` are two separate libraries that shouldn't technically coexist. The latter is supposed to replace the first one. So I'm not sure this CMakeLists.txt actually has a parent (or I'm not sure what you mean by parent).